README.md

```diff
@@ -1,4 +1,4 @@
-MPAS-v6.3
+MPAS-v7.0
```

Comment:
Just a note: this change was already on ocean/develop before this PR (from a framework merge), but it shows up here again. Same with all the atmospheric files below. It is strange that they show up here, because they don't show up on a git diff:
`git diff origin/ocean/develop origin/ocean/coastal --stat|grep README.md`
produces nothing. I don't understand it.